src/python/bindings/bind.cpp

```diff
@@ -112,9 +111,11 @@ PYBIND11_MODULE(smartredisPy, m) {
         .def("poll_list_length_gte", &PyClient::poll_list_length_gte)
         .def("poll_list_length_lte", &PyClient::poll_list_length_lte)
         .def("get_datasets_from_list", &PyClient::get_datasets_from_list)
-        .def("get_dataset_list_range", &PyClient::get_dataset_list_range);
+        .def("get_dataset_list_range", &PyClient::get_dataset_list_range)
+        .def("to_string", &PyClient::to_string);
 
     // Python Dataset class
+    #define DATASET_METHOD(name) def(#name, &PyClient::name)
```

Comment:
Same "macro used" question here

Reply:
Not yet -- it will be a separate PR to switch over to it